src/Provider/PackageArchiver.php

```diff
@@ -145,12 +153,12 @@ public function archive_updates( $value ) {
 			$slug = $update_data[ $type ];
 
 			// Bail if the package isn't whitelisted.
-			if ( ! $manager->has_package( $slug, $type ) ) {
+			if ( ! $this->satispress->contains( [ 'slug' => $slug, 'type' => $type ] ) ) {
 				continue;
 			}
 
 			try {
-				$package = $manager->get_package( $slug, $type );
+				$package = $this->packages->first_where( [ 'slug' => $slug, 'type' => $type ] );
```

Comment:
Following on from the above comment, `$this->packages` could be renamed as `$this->all_packages`.